deploy_local_testnet.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -i
+
+
+echo "0. Changing working directory as home folder"
+cd ~
+
+echo "1. Updating apt-get"
+sudo apt-get update -qq
+
+echo "2. Installing basic essentials (might take some time)"
+sudo apt-get install wget git build-essential -y -qq > /dev/null
+
+echo "3. Installing golang v1.18"
+wget -q -O - https://git.io/vQhTU | bash -s - --version 1.18 > /dev/null 2>&1
```

Comment:
https://git.io/vQhTU resolves to https://raw.githubusercontent.com/canha/golang-tools-install-script/master/goinstall.sh which:
* is unpredictable because pointing to `master`
* depending on `canha`
* uselessly depending (and making it less transparent) on `git.io` website.